extensions/wikia/MercuryApi/MercuryApiHooks.class.php

```diff
@@ -4,6 +4,7 @@ class MercuryApiHooks {
 
 	const SERVICE_API_BASE = '/api/v1/';
 	const SERVICE_API_ARTICLE = 'article/';
+	const SERVICE_API_CURATED_CONTENT = 'curatedContent/';
```

Comment:
Yup. The URLs I can think of now we have to purge are:
* Ember calls to Hapi:
  * the ones you mentioned above,
* Hapi calls to MediaWiki:
  * `/wikia.php?controller=MercuryApi&method=getArticle&title=${Main_Page}` https://github.com/Wikia/mercury/blob/dev/server/lib/MediaWiki.ts#L92, 
  * all of `/wikia.php?controller=MercuryApi&method=getCuratedContentSection&section=${section}` https://github.com/Wikia/mercury/blob/dev/server/lib/MediaWiki.ts#L118,
  * all of `/wikia.php?controller=ArticlesApi&method=getList&…&category=${categoryName}` https://github.com/Wikia/mercury/blob/dev/server/lib/MediaWiki.ts#L128,
* first static pages generated by Hapi e.g.:
  * http://elderscrolls.wikia.com/wiki/The_Elder_Scrolls_Wiki (do we need http://elderscrolls.wikia.com/ too?),
  * http://elderscrolls.wikia.com/main/section/TES%2520Online,
  * http://elderscrolls.wikia.com/main/category/Skyrim.
* native apps calls:
  * `/wikia.php?controller=ArticlesApi&method=getList` which should remain cached and purged,
  * all of `/wikia.php?controller=ArticlesApi&method=getList&section=${section}` which caching and purging is already added in this PR.
  * I'm wondering how apps get category entries and how it's cached/purged if at all.

…so much has to be added and/or rewritten. :wrench: